portfolio/src/main/webapp/galleryScript.js

```diff
@@ -0,0 +1,20 @@
+// Functions for the slideshows in Gallery
+var slideIndex = 1;
+displaySlide(slideIndex);
+
+function moveSlide(n) {
```

Comment:
nit: perhaps use a name like `delta` instead of `n` to make it clear that `n` is not an index

Reply:
Got it! Changing n to be called moveFactor to clarify that it's meant to show how much slideIndex is supposed to move.